src/utils/requests.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from "@angular/core";
+import { Headers, Http } from "@angular/http";
+
+@Injectable()
+export class Requests {
+
+	private headers = new Headers({'Content-Type': 'application/json'});
+	
+	constructor(private http: Http) { }
```

Comment:
Maybe constructor should take a set of 'Additional Headers', so we can e.g. pass in the Authentication Token to be used.